api/sdk.api.md

```diff
@@ -52,6 +52,7 @@ export class Confidence implements EventSender, Trackable, FlagResolver {
 // @public
 export interface ConfidenceOptions {
     clientSecret: string;
+    customResolveBaseUrl?: string;
```

Comment:
Could we change the option to `resolverBaseUrl`? I'm thinking "custom" is a little redundant in this context. And resolve vs resolver, I'm not sure, but "resolve" is the action, whereas resolver more signifies the service/resource which I think feels more appropriate..

Reply:
Is it redundant though? My thinking is it's "custom" because you might want to specify it, but if you don't we use a standard url 🤔